rosbag2_transport/test/rosbag2_transport/test_play_for.cpp

```diff
@@ -22,7 +22,10 @@
 #include <unordered_map>
```

Comment:
Clang tidy shows that we are not using 
```
#include <string>
#include <unordered_map>
```
And they could be deleted as well.

Actually clang tidy shows up that whole block of includes
```cpp
//#include "rclcpp/experimental/buffers/intra_process_buffer.hpp"
//#include "rclcpp/node.hpp"
//#include "rclcpp/qos_event.hpp"
//
//#include "rosbag2_test_common/subscription_manager.hpp"
//
//#include "rosbag2_transport/player.hpp"
//#include "rosbag2_transport/qos.hpp"
```
could be deleted. 
We either not using them or have included them from somewhere else.